tracer/src/Datadog.Trace/Iast/Telemetry/ExecutedTelemetryHelper.cs

```diff
@@ -128,20 +128,21 @@ private string GetExecutedSourceTag(IastInstrumentedSources source)
     private string GetExecutedSinkTag(IastInstrumentedSinks vulnerability)
         => vulnerability switch
         {
-            IastInstrumentedSinks.LdapInjection => BasicExecutedTag + SinkExecutedTag + "ldap_injection",
-            IastInstrumentedSinks.SqlInjection => BasicExecutedTag + SinkExecutedTag + "sql_injection",
-            IastInstrumentedSinks.CommandInjection => BasicExecutedTag + SinkExecutedTag + "command_injection",
-            IastInstrumentedSinks.InsecureCookie => BasicExecutedTag + SinkExecutedTag + "insecure_cookie",
-            IastInstrumentedSinks.NoHttpOnlyCookie=> BasicExecutedTag + SinkExecutedTag + "no_http_only_cookie",
-            IastInstrumentedSinks.NoSameSiteCookie => BasicExecutedTag + SinkExecutedTag + "no_samesite_cookie",
             IastInstrumentedSinks.WeakCipher => BasicExecutedTag + SinkExecutedTag + "weak_cipher",
             IastInstrumentedSinks.WeakHash => BasicExecutedTag + SinkExecutedTag + "weak_hash",
+            IastInstrumentedSinks.SqlInjection => BasicExecutedTag + SinkExecutedTag + "sql_injection",
+            IastInstrumentedSinks.CommandInjection => BasicExecutedTag + SinkExecutedTag + "command_injection",
             IastInstrumentedSinks.PathTraversal => BasicExecutedTag + SinkExecutedTag + "path_traversal",
+            IastInstrumentedSinks.LdapInjection => BasicExecutedTag + SinkExecutedTag + "ldap_injection",
             IastInstrumentedSinks.Ssrf => BasicExecutedTag + SinkExecutedTag + "ssrf",
             IastInstrumentedSinks.UnvalidatedRedirect => BasicExecutedTag + SinkExecutedTag + "unvalidated_redirect",
+            IastInstrumentedSinks.InsecureCookie => BasicExecutedTag + SinkExecutedTag + "insecure_cookie",
+            IastInstrumentedSinks.NoHttpOnlyCookie=> BasicExecutedTag + SinkExecutedTag + "no_http_only_cookie",
+            IastInstrumentedSinks.NoSameSiteCookie => BasicExecutedTag + SinkExecutedTag + "no_samesite_cookie",
             IastInstrumentedSinks.WeakRandomness => BasicExecutedTag + SinkExecutedTag + "weak_randomness",
             IastInstrumentedSinks.HardcodedSecret => BasicExecutedTag + SinkExecutedTag + "hardcoded_secret",
             IastInstrumentedSinks.XContentTypeHeaderMissing => BasicExecutedTag + SinkExecutedTag + "xcontenttype_header_missing",
+            IastInstrumentedSinks.TrustBoundaryViolation => BasicExecutedTag + SinkExecutedTag + "trust_boundary_violation",
```

Comment:
not really an issue, but I wondered why you reordered this?

Reply:
To match the order of the enum values (so, new ones should go last). It was making my eyes sore a bit... 😅